e2e-tests/src/test/java/dev/streamx/cli/AppTest.java

```diff
@@ -0,0 +1,78 @@
+package dev.streamx.cli;
+
+
+import dev.streamx.cli.commandline.*;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+
+
+public class AppTest 
+{
+
+    private static final String STREAMX_URL = "http://localhost:8081/";
```

Comment:
rename STREAMX_URL to WEB_DELIVERY_PORT_URL

You can extract to e.g. @ConfigProperty(name = "streamx.cli.e2e.web-delivery.url")